src/utils/elements/getElementsForUpsert.ts

```diff
@@ -10,6 +10,9 @@ type RawElementValue = string | string[] | number | undefined;
 const getElementValue = (value: RawElementValue, element: ContentTypeElements.ContentTypeElementModel): ElementValue => {
   switch (element.type) {
     case 'rich_text': {
+      if (typeof value !== 'string') {
+        return undefined;
+      }
       if (typeof value === 'string' && value.trim().startsWith('<')) {
```

Comment:
`typeof value === 'string'` is unnecessary here, as you control whether the value is not a string in the condition above